### PR TITLE
fix: correct last modified date for drag and drop PE-674

### PR DESCRIPTION
**Pull request by jdaev**

### CODE REVIEW

**Review by arielmelendez — APPROVED** [review]

File: lib/pages/drive_detail/components/drive_file_drop_zone.dart [hunk 1]
```diff
@@ -71,12 +71,20 @@ class _DriveFileDropZoneState extends State<DriveFileDropZone> {
       final fileName = await controller.getFilename(htmlFile);
       final fileMIME = await controller.getFileMIME(htmlFile);
       final fileLength = await controller.getFileSize(htmlFile);
+      var fileLastModified;
+      // In case of browsers that don't support it. Fails on firefox for now
+      try {
+        fileLastModified = await controller.getFileLastModified(htmlFile);
```
Comment (matibat) [inline lib/pages/drive_detail/components/drive_file_drop_zone.dart:77]: Is this line te one that could fail? Why so? What's different on Firefox?
```ts
        fileLastModified = await controller.getFileLastModified(htmlFile);
```

Comment (matibat) [reply lib/pages/drive_detail/components/drive_file_drop_zone.dart:77]: The workaround is to set it to `now()`. Does it have any unintended side effects?

File: lib/pages/drive_detail/components/drive_file_drop_zone.dart [hunk 2]
```diff
@@ -71,12 +71,20 @@ class _DriveFileDropZoneState extends State<DriveFileDropZone> {
...
+      } catch (e) {
+        fileLastModified = DateTime.now();
+      }
+
       final htmlUrl = await controller.createFileUrl(htmlFile);
       final fileToUpload = XFile(
         htmlUrl,
         name: fileName,
         mimeType: fileMIME,
-        lastModified: DateTime.now(),
```
Comment (matibat) [inline lib/pages/drive_detail/components/drive_file_drop_zone.dart:79]: Ohh so it was always set to `now()` before. I see

**Review by matibat — COMMENTED** [review]
It's looking good. Left one single question there

File: lib/pages/drive_detail/components/drive_file_drop_zone.dart (hunk 1 above)
Comment (jdaev) [reply lib/pages/drive_detail/components/drive_file_drop_zone.dart:77]: It was the earlier behavior. No difference from prod.

File: lib/pages/drive_detail/components/drive_file_drop_zone.dart (hunk 2 above)
Comment (jdaev) [reply lib/pages/drive_detail/components/drive_file_drop_zone.dart:79]: Yeah, it was.

**Comment by fedellen** [comment]
Looking good here -- as I understand this PR is currently blocked by some UI issues with the Flutter version bumps

**Review by matibat — APPROVED** [review]
LGTM :+1:

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**